modules/rules_bison/0.2.2/MODULE.bazel

```diff
@@ -0,0 +1,45 @@
+module(
+    name = "rules_bison",
+    version = "0.2.2",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "rules_m4", version = "0.2.3")
+
+bazel_dep(
+    name = "googletest",
+    version = "1.12.1",
+    dev_dependency = True,
+    repo_name = "com_google_googletest",
+)
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.2.1",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "stardoc",
+    version = "0.5.3",
+    dev_dependency = True,
+    repo_name = "io_bazel_stardoc",
+)
+
+default_toolchain = use_extension(
+    "//bison/internal:default_toolchain_ext.bzl",
+    "default_toolchain_ext",
+)
+use_repo(default_toolchain, "bison")
+
+register_toolchains("@bison//:toolchain")
+
+testutil = use_extension(
+    "//bison/internal:testutil_ext.bzl",
+    "rules_bison_testutil_ext",
+    dev_dependency = True,
+)
+use_repo(testutil, "rules_bison_testutil")
+
+register_toolchains(
+    "@rules_bison_testutil//toolchains:all",
+    dev_dependency = True,
```

Comment:
This parameter requires Bazel 6.2.0 or higher. Could you add a `bazel_compatibility` specification to `module`? https://bazel.build/rules/lib/globals/module#module has the details.

Reply:
Done.